// @adobe/react-spectrum-workflow/dist/Deduplication.js
/**
Copyright 2024 Adobe. All rights reserved.
This file is licensed to you under the Apache License, Version 2.0 (the "License");
you may not use this file except in compliance with the License. You may obtain a copy
of the License at http://www.apache.org/licenses/LICENSE-2.0

Unless required by applicable law or agreed to in writing, software distributed under
the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
OF ANY KIND, either express or implied. See the License for the specific language
governing permissions and limitations under the License.
**/
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.A4uDeduplication = A4uDeduplication;

var _react = _interopRequireDefault(require("react"));

function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { "default": obj }; }

function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }

function A4uDeduplication(_ref) {
  var props = _extends({}, _ref);

  return /*#__PURE__*/_react["default"].createElement("svg", _extends({
    viewBox: "0 0 36 36"
  }, props, props), /*#__PURE__*/_react["default"].createElement("circle", {
    fillRule: "evenodd",
    cx: "7.939",
    cy: "6.25",
    r: "3.75"
  }), /*#__PURE__*/_react["default"].createElement("polygon", {
    fillRule: "evenodd",
    points: "21.506 10 12.756 10 17.131 2.5 21.506 10"
  }), /*#__PURE__*/_react["default"].createElement("circle", {
    fillRule: "evenodd",
    cx: "11.939",
    cy: "30.25",
    r: "3.75"
  }), /*#__PURE__*/_react["default"].createElement("polygon", {
    fillRule: "evenodd",
    points: "27.603 34 18.853 34 23.228 26.5 27.603 34"
  }), /*#__PURE__*/_react["default"].createElement("polygon", {
    fillRule: "evenodd",
    points: "32.121 10 23.371 10 27.746 2.5 32.121 10"
  }), /*#__PURE__*/_react["default"].createElement("path", {
    fillRule: "evenodd",
    d: "M27.939,12.058h-20V13.28a1.514,1.514,0,0,0,.723,1.3L14.351,18.6a3.056,3.056,0,0,1,1.114,2.377V25.17a.733.733,0,0,0,.714.75H19.7a.733.733,0,0,0,.714-.75V20.976A3.056,3.056,0,0,1,21.527,18.6l5.689-4.015a1.514,1.514,0,0,0,.723-1.3Z"
  }));
}
