// @adobe/react-spectrum-workflow/dist/FlashOff.js
/**
Copyright 2024 Adobe. All rights reserved.
This file is licensed to you under the Apache License, Version 2.0 (the "License");
you may not use this file except in compliance with the License. You may obtain a copy
of the License at http://www.apache.org/licenses/LICENSE-2.0

Unless required by applicable law or agreed to in writing, software distributed under
the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
OF ANY KIND, either express or implied. See the License for the specific language
governing permissions and limitations under the License.
**/
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.A4uFlashOff = A4uFlashOff;

var _react = _interopRequireDefault(require("react"));

function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { "default": obj }; }

function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }

function A4uFlashOff(_ref) {
  var props = _extends({}, _ref);

  return /*#__PURE__*/_react["default"].createElement("svg", _extends({
    viewBox: "0 0 36 36"
  }, props, props), /*#__PURE__*/_react["default"].createElement("polygon", {
    fillRule: "evenodd",
    points: "13.823 20.473 8 36 8.9 36 18.393 25.065 13.823 20.473"
  }), /*#__PURE__*/_react["default"].createElement("polygon", {
    fillRule: "evenodd",
    points: "18.26 13.609 26 2 12 2 10.714 6.026 18.26 13.609"
  }), /*#__PURE__*/_react["default"].createElement("polygon", {
    fillRule: "evenodd",
    points: "23.643 19.019 28 14 18.649 14 23.643 19.019"
  }), /*#__PURE__*/_react["default"].createElement("polygon", {
    fillRule: "evenodd",
    points: "7.976 14.598 6.25 20 13.352 20 7.976 14.598"
  }), /*#__PURE__*/_react["default"].createElement("rect", {
    fillRule: "evenodd",
    height: "43.854",
    rx: "0.818",
    ry: "0.818",
    transform: "translate(-8.163 18.293) rotate(-45)",
    width: "2.455",
    x: "16.773",
    y: "-2.926"
  }));
}
