// @adobe/react-spectrum-workflow/dist/BidRule.js
/**
Copyright 2024 Adobe. All rights reserved.
This file is licensed to you under the Apache License, Version 2.0 (the "License");
you may not use this file except in compliance with the License. You may obtain a copy
of the License at http://www.apache.org/licenses/LICENSE-2.0

Unless required by applicable law or agreed to in writing, software distributed under
the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
OF ANY KIND, either express or implied. See the License for the specific language
governing permissions and limitations under the License.
**/
"use strict";

Object.defineProperty(exports, "__esModule", {
  value: true
});
exports.A4uBidRule = A4uBidRule;

var _react = _interopRequireDefault(require("react"));

function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { "default": obj }; }

function _extends() { _extends = Object.assign ? Object.assign.bind() : function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; return _extends.apply(this, arguments); }

function A4uBidRule(_ref) {
  var props = _extends({}, _ref);

  return /*#__PURE__*/_react["default"].createElement("svg", _extends({
    viewBox: "0 0 36 36"
  }, props, props), /*#__PURE__*/_react["default"].createElement("rect", {
    fillRule: "evenodd",
    height: "8.485",
    transform: "translate(-1.456 20.485) rotate(-45)",
    width: "8.485",
    x: "19.757",
    y: "7.757"
  }), /*#__PURE__*/_react["default"].createElement("rect", {
    fillRule: "evenodd",
    height: "3.155",
    rx: "0.789",
    ry: "0.789",
    transform: "translate(-3.946 26.491) rotate(-44.995)",
    width: "12.619",
    x: "23.7",
    y: "16.432"
  }), /*#__PURE__*/_react["default"].createElement("rect", {
    fillRule: "evenodd",
    height: "3.155",
    rx: "0.789",
    ry: "0.789",
    transform: "translate(1.019 14.507) rotate(-44.995)",
    width: "12.619",
    x: "11.713",
    y: "4.445"
  }), /*#__PURE__*/_react["default"].createElement("path", {
    fillRule: "evenodd",
    d: "M4.06,34.06,1.94,31.94a1.5,1.5,0,0,1,0-2.122L18,15l3,3L6.182,34.06A1.5,1.5,0,0,1,4.06,34.06Z"
  }), /*#__PURE__*/_react["default"].createElement("path", {
    fillRule: "evenodd",
    d: "M34,30V29a1,1,0,0,0-1-1H23a1,1,0,0,0-1,1v1H20.5a.5.5,0,0,0-.5.5v3a.5.5,0,0,0,.5.5h15a.5.5,0,0,0,.5-.5v-3a.5.5,0,0,0-.5-.5Z"
  }));
}
